### PR TITLE
fix(proto): use buf, like we do on dymension repo

### PR DESCRIPTION
**Pull request by danwt**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

For Author:

- [ ]  Targeted PR against correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to Github issue with discussion and accepted design
- [ ]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,21 +33,119 @@ test_evm: ## Run go test
...
+protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
+containerProtoGen=cosmos-sdk-proto-gen-$(protoVer)
+containerProtoFmt=cosmos-sdk-proto-fmt-$(protoVer)
+# ------
+# NOTE: cosmos/proto-builder image is needed because clang-format is not installed
+#       on the tendermintdev/sdk-proto-gen docker image.
+#		Link to the cosmos/proto-builder docker images:
+#       https://github.com/cosmos/cosmos-sdk/pkgs/container/proto-builder
+#
+protoCosmosVer=0.11.2
```
Comment (hoangdv2429) [inline Makefile:54]: can we bump this to use 0.14

Comment (danwt) [reply Makefile:54]: we decided to followup in other prs

**Review by mtsitrin — APPROVED** [review]

**Review by hoangdv2429 — APPROVED** [review]

**Comment by 0xbcdev** [comment]
@danwt the Makefile missed setting version to variables:
https://github.com/dymensionxyz/dymension/blob/983dd904dacc3b5a6ff5c3122b02fdbf450cd730/Makefile#L13-L20

**Comment by danwt** [comment]
Thanks for pointing that out I was in a rush
I'll fix it

**Pull request merged**